MediaBrowser.Controller/MediaEncoding/EncodingHelper.cs

```diff
@@ -2205,7 +2208,11 @@ public bool CanStreamCopyVideo(EncodingJobInfo state, MediaStream videoStream)
                     return false;
                 }
 
-                if (!requestedRangeTypes.Contains(videoStream.VideoRangeType.ToString(), StringComparison.OrdinalIgnoreCase))
+                // DOVIWithHDR10 should be compatible with HDR10 supporting players. Same goes with HLG and of course SDR. So allow copy of those formats
+                if (!requestedRangeTypes.Contains(videoStream.VideoRangeType.ToString(), StringComparison.OrdinalIgnoreCase)
+                     && !((requestedRangeTypes.Contains("HDR10", StringComparison.OrdinalIgnoreCase) && videoStream.VideoRangeType == VideoRangeType.DOVIWithHDR10)
+                            || (requestedRangeTypes.Contains("HLG", StringComparison.OrdinalIgnoreCase) && videoStream.VideoRangeType == VideoRangeType.DOVIWithHLG)
+                            || (requestedRangeTypes.Contains("SDR", StringComparison.OrdinalIgnoreCase) && videoStream.VideoRangeType == VideoRangeType.DOVIWithSDR)))
```

Comment:
```suggestion
                var haveDOVIWithHDR10 = requestedRangeTypes.Contains(VideoRangeType.DOVIWithHDR10.ToString(), StringComparison.OrdinalIgnoreCase);
                var haveDOVIWithHLG = requestedRangeTypes.Contains(VideoRangeType.DOVIWithHLG.ToString(), StringComparison.OrdinalIgnoreCase);
                var haveDOVIWithSDR = requestedRangeTypes.Contains(VideoRangeType.DOVIWithSDR.ToString(), StringComparison.OrdinalIgnoreCase);

                // DOVIWithHDR10 should be compatible with HDR10 supporting players. Same goes with HLG and of course SDR. So allow copy of those formats
                if (!requestedRangeTypes.Contains(videoStream.VideoRangeType.ToString(), StringComparison.OrdinalIgnoreCase)
                    && !((videoStream.VideoRangeType == VideoRangeType.DOVIWithHDR10 && haveDOVIWithHDR10)
                          || (videoStream.VideoRangeType == VideoRangeType.DOVIWithHLG && haveDOVIWithHLG)
                          || (videoStream.VideoRangeType == VideoRangeType.DOVIWithSDR && haveDOVIWithSDR)))
```

Each line is already too long, Extract the contains* to vars to increase readability.